equistore-torch/src/labels.cpp

```diff
@@ -51,32 +51,35 @@ static torch::Tensor initializer_list_to_tensor(
     return tensor.reshape({static_cast<int64_t>(count), static_cast<int64_t>(size)});
 }
 
-static std::vector<std::string> normalize_names(torch::IValue names) {
+std::vector<std::string> equistore_torch::details::normalize_names(torch::IValue names, std::string context) {
```

Comment:
Will there be doc for this? To what do the names need to be normalized? First thought the context has to do something with the context manager of torch, but it is just for error context. Maybe we can also name it this way (e.g. `error_msg_prefix`)?